app/src/components/Map/Timebar.jsx

```diff
@@ -467,6 +471,13 @@ class Timebar extends Component {
   }
 
   render() {
+    const dateFormat = 'DD MMM YYYY';
+    const startDateText = isMobile.phone || isMobile.tablet ? ' start' : 'start date';
```

Comment:
Not sure I understand, this changes the text content to adapt to smaller viewports right? So something like this should work just the same:
```
.whatever {
  &::before {
    content: "start";
  }
  @media #{$mq-mobile-large} {
    &::before {
      content: "start date";
    }
  }
}
```

This is a standard responsive design use case, the lib you are using is intended for redirections/change of feature set depending on device which is different.
Or is there something I'm missing?